recipes/xorg-libximerna/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.1.4" %}
+
+package:
+  name: xorg-libximerna
+  version: {{ version }}
+
+source:
+  url: https://gitlab.freedesktop.org/xorg/lib/libxinerama/-/archive/libXinerama-{{ version }}/libxinerama-libXinerama-{{ version }}.tar.gz
+  sha256: d9fb7f224281f5467d36f54b01a8f49e866a994410d5bce3b019f2da8bf8e643
+
+build:
+  number: 1
```

Comment:
The build number should start at 0.

Reply:
Sorry, this was adapted from my personal scripts.